doc/book/application-integration/usage-in-a-laminas-mvc-application.md

```diff
@@ -0,0 +1,106 @@
+# Usage in a laminas-mvc Application
+
+The following example shows _one_ potential use case of laminas-eventmanager within
+a laminas-mvc based application.
+The example creates a listener for logging errors of an application with [laminas-log](https://docs.laminas.dev/laminas-log/).
+
+Before starting, make sure laminas-log is [installed and configured](https://docs.laminas.dev/laminas-log/installation/).
+
+laminas-eventmanager is already present in laminas-mvc based applications as it is an event-driven MVC layer based on the event manager.
+
+## Create Listener
+
+Create a listener as class using [a listener aggregate](../aggregates.md) and add the logger via constructor injection to create a valid object with all required dependencies, e.g. `module/Application/src/Listener/ErrorListener.php`:
+
+```php
+namespace Application\Listener;
+
+use Exception;
+use Laminas\EventManager\AbstractListenerAggregate;
+use Laminas\EventManager\EventManagerInterface;
+use Laminas\Log\Logger;
+use Laminas\Mvc\MvcEvent;
+
+final class ErrorListener extends AbstractListenerAggregate
+{
+    public function __construct(
+        private readonly Logger $logger
+    ) {}
+
+    public function attach(EventManagerInterface $events, $priority = 1): void
+    {
+        $this->listeners[] = $events->attach(
+            MvcEvent::EVENT_DISPATCH_ERROR,
+            [$this, 'onDispatchError']
+        );
+    }
+
+    public function onDispatchError(MvcEvent $event): void
+    {
+        /** @var Exception|null $exception */
+        $exception = $event->getParam('exception');
+        if ($exception) {
+            $this->logger->crit('Error: ' . $exception->getMessage());
+        } else {
+            $this->logger->crit('Error: ' . $event->getError());
+        }
+    }
+}
+```
+
+The listener aggregate was chosen because [it can listen to even more events](../aggregates.md#recommendations) due to the possible extension.
```

Comment:
```suggestion
The listener aggregate was chosen because [to allow it to listen to additional events](../aggregates.md#recommendations) if we want to later.
```